### PR TITLE
Support tabs on scenario detail page.

### PR DESCRIPTION
**Pull request by SoumyaMahadevan**
## Description of the change
Added tabs for scenario detail page. "Projections" tab is selected by default. The second tab is blank and not yet implemented. PR #202 
image
image

## Type of change

- [ ] Bug fix (non-breaking change that fixes an issue)
- [X] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Configuration change (adjusts configuration to achieve some end related to functionality, development, performance, or security)

## Related issues

Part of #202 

### Development

These boxes should be checked by the submitter prior to merging:

- [X] Manual testing has been performed locally

### Code review

These boxes should be checked by reviewers prior to merging:

- [x] This pull request has a descriptive title and information useful to a reviewer
- [x] This pull request has been moved out of a Draft state, has no "Work In Progress" label, and has assigned reviewers
- [x] Potential security implications or infrastructural changes have been considered, if relevant

### CODE REVIEW

**Comment by SoumyaMahadevan** [comment]
(1) Fixed alignment between textlabel and green border.
(2) Added feature flag to turn off rate of spread panel as necessary.

image

File: src/page-multi-facility/MultiFacilityImpactDashboard.tsx [hunk 1]
```diff
@@ -96,6 +96,8 @@ const ScenarioPanels: React.FC<ScenarioPanelsProps> = (props) => {
   return <ScenarioPanelsDiv>{panelChildren}</ScenarioPanelsDiv>;
 };
 
+// Boolean to switch off "Rate of Projections" tab if necessary
```
Comment (jessex) [inline src/page-multi-facility/MultiFacilityImpactDashboard.tsx:99]: I think this comment is in the wrong place?

File: src/feature-flags/featureFlags.tsx [hunk 2]
```diff
@@ -12,6 +13,7 @@ export const FeatureFlagsProvider: React.FC = ({ children }) => {
     <FlagsProvider
       flags={{
         useRt: process.env.GATSBY_USE_RT === "true",
+        showRateOfSpreadTab: true,
```
Comment (jessex) [inline src/feature-flags/featureFlags.tsx:16]: Let's add a TODO on this to remove the feature flag once #202 has been completed.

File: src/page-multi-facility/MultiFacilityImpactDashboard.tsx (hunk 1 above)
Comment (SoumyaMahadevan) [reply src/page-multi-facility/MultiFacilityImpactDashboard.tsx:99]: Oops yeah. I had the boolean flag within the file before I found feature flags.

**Pull request merged**